src/components/PageMeta.tsx

```diff
@@ -11,7 +11,7 @@ export function PageMeta(props: IProps) {
   return (
     <Head>
       <title>{title}</title>
-      <meta property="og:title" content={description} key="title" />
+      {description && <meta property="og:title" content={description} key="title" />}
```

Comment:
Fjerner obligatorisk description her siden jeg så at de fleste sidene ikke har en apptekst for beskrivelse som finnes i Sanity. 😅 Skal sannsynligvis fjerne de som mangler i neste uke, men må få en avklaring på dette først. Det kan jo hende vi ønsker å skrive beskrivelser for alle sidene i stedet.